formula/src/test/java/com/instacart/formula/subjects/DuplicateListenerKeysHandledByIndexing.kt

```diff
@@ -10,7 +10,7 @@ object DuplicateListenerKeysHandledByIndexing {
 
     fun test(runtime: TestableRuntime) = runtime.test(TestFormula(), Unit)
 
-    class TestOutput(
+    data class TestOutput(
```

Comment:
A bunch of places were captured by the validation logic which didn't provide equality implementation of the output.